extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php

```diff
@@ -64,6 +65,9 @@ public static function onArticleSaveComplete(
 	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 		\Wikia::addAssetsToOutput( 'community_page_benefits_js' );
 		\Wikia::addAssetsToOutput( 'community_page_benefits_scss' );
+		\Wikia::addAssetsToOutput( 'community_page_new_user_modal_js' );
+		\Wikia::addAssetsToOutput( 'community_page_new_user_modal_scss' );
```

Comment:
Do both of these need to be loaded on every single page view or only when the cookie is set? Also, from the ticket, it looks like this is only for logged in users, so can the assets be restricted to only load for logged-in users?